.github/workflows/check-users-in-space-developer-role.yml

```diff
@@ -0,0 +1,70 @@
+on:
+  workflow_dispatch:
+    inputs:
+      CF_SPACE:
+        description: PaaS space to target
+        default: 'bat-prod'
+        required: true
+jobs:
+
+  CF:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: DFE-Digital/github-actions/setup-cf-cli@master
+        with:
+          CF_USERNAME: ${{ secrets.CF_USERNAME }}
+          CF_PASSWORD: ${{ secrets.CF_PASSWORD }}
+          CF_SPACE_NAME: bat-qa
+          # Optional inputs
+          CF_CLI_VERSION: v7
+          CF_ORG_NAME: dfe
+          CF_API_URL:  https://api.london.cloud.service.gov.uk
+          INSTALL_CONDUIT: true
+
+      - name : Install cf client
+        id: cf_space
+        shell: pwsh
+        run: |
+          $space= "${{ github.event.inputs.CF_SPACE }}"
```

Comment:
add a checkout action and invoke `script/check-users-in-space-developer-role.ps1`